datasets/bnl_newspapers/README.md

```diff
@@ -0,0 +1,206 @@
+---
+annotations_creators:
+- no-annotation
+language_creators:
+- found
+languages:
+- ar
+- da
+- de
+- fi
+- fr
+- lb
+- nl
+- pt
+licenses:
+- cc0-1.0
+multilinguality:
+- multilingual
+pretty_name: BnL newspapers
+size_categories:
+- 100K<n<1M
+source_datasets:
+- original
+task_categories:
+- sequence-modeling
+task_ids:
+- language-modeling
+---
+
+# Dataset Card for bnl-processed-newspapers 
+
+## Table of Contents
+- [Table of Contents](#table-of-contents)
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:**: [https://data.bnl.lu/data/historical-newspapers/](https://data.bnl.lu/data/historical-newspapers/) 
+- **Repository:**
+- **Paper:**
+- **Leaderboard:**
+- **Point of Contact:**
+
+### Dataset Summary
+
+The BnL has digitised over 800.000 pages of Luxembourg newspapers. This dataset contains a subset of these newspapers which sit under the "Processed Datasets" collection. The BNL:
+
+> processed all newspapers and monographs that are in the public domain and extracted the full text and associated meta data of every single article, section, advertisement… The result is a large number of small, easy to use XML files formatted using Dublin Core.
+
+This dataset contains the newspapers from these processed collections. 
+
+[More Information Needed]
```

Comment:
I guess this section needs rephrasing...

Reply:
I have updated this to reflect the current configuration. I still hope to add the other collections as different configurations so I can update this further in a future pull request if that's okay?